addons/website/static/src/scss/website.backend.scss

```diff
@@ -337,27 +337,44 @@
         width:100%;
     }
 
-    .o_field_iframe{
+    .o_field_iframe {
         width: 100%;
         height: 100%;
+
         div.is_mobile {
             @include media-breakpoint-up(md) {
-                // mobile frame is rounded
-                margin: auto !important;
-                padding: 53px 11px 58px 28px;
-                width: 416px;
-                iframe {
-                    height: 735px;
-                    border-radius: 15px;
-                }
-                .img_mobile {
-                    pointer-events: none;
-                    display: block;
-                    position: absolute;
-                    top: 16px;
-                    left: calc(50% - 200px)
-                }
+                position: relative;
+            }
+        }
+    }
+}
+
+// Mobile preview
+.o_website_preview.o_is_mobile .o_iframe_container, .o_view_form_theme_preview_controller .o_field_iframe div.is_mobile {
```

Comment:
Doing this below your rules is not what I meant... it would be better as you did before otherwise. The point here is to add the `@extend` calls in the already existing nested rules (where you define the rules for `o_website_preview` and `o_field_iframe`.

```scss
// Mobile preview in the website builder.
.o_website_preview.o_is_mobile .o_iframe_container {
    @extend %o-mobile-phone;
}

// Mobile preview in the theme preview page.
.o_view_form_theme_preview_controller .o_field_iframe div.is_mobile {
    @extend %o-mobile-phone;
}
```

Reply:
it's done but I'm not 100% sure where to put the rule `%o-mobile-phone`

Reply:
Should be ok where you put it :+1: